packaging/nnfw.spec

```diff
@@ -58,6 +59,10 @@ BuildRequires:  libjpeg-devel
 BuildRequires:  gtest-devel
 %endif
 
+%if %{trix_support} == 1
+BuildRequires:  npu-engine-devel
+%endif
```

Comment:
We need `nnfw.spec` update for both loader and backend as fast as possible. Could you make PR for this only?

Reply:
I remove this update in this PR :)